src/mlpack/methods/softmax_regression/softmax_regression.hpp

```diff
@@ -141,13 +140,8 @@ class SoftmaxRegression
    * @param numClasses Number of classes for classification.
    * @return Objective value of the final point.
    */
-  double Train(const arma::mat &data, const arma::vec& labels,
-               const size_t numClasses);
-
-  //! Sets the size of the input vector.
-  size_t& InputSize() { return inputSize; }
-  //! Gets the size of the input vector.
-  size_t InputSize() const { return inputSize; }
```

Comment:
Sure, I'll go ahead and merge this and change `InputSize()` to `FeatureSize()`.  It should be `n_rows`, not `n_cols` -- Armadillo matrices are column-major, so each column corresponds to an observation and each row corresponds to a feature.